src/status.js

```diff
@@ -0,0 +1,25 @@
+function updateTasks() {
+  const list = document.getElementsByTagName('li');
+  const tempTasks = [];
+
+  if (list.length !== 0) {
+    Array.from(list).forEach((li, index) => {
+      const div = li.getElementsByTagName('div')[0];
+      const completed = div.getElementsByTagName('input')[0].checked;
+      const id = div.getElementsByTagName('input')[0].name;
+      const description = div.getElementsByTagName('input')[1].value;
+
+      const task = {
+        completed,
+        description,
+        index,
+        id,
+      };
+      tempTasks.push(task);
+    });
+  }
+
+  return tempTasks;
+}
```

Comment:
- [ ] I suggest that you add an array of objects that will be rendered to the DOM when the page load.. currently I can add a task but it doesn't get posted to the screen so that I can be able to test it
- [ ] In this status file you need to add methods related to the status updates (completed: true / false). please work on that 
- [ ] The task object's values for `index` and `completed` are supposed to be updated upon user actions.